website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
 RISC Zero provides an already deployed verifier contract in the Sepolia network for your convenience.
 You can choose to use this contract or deploy your own.
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
```

Comment:
Do we want a column here indicating the control root and/or list of allowed programs? 
Or perhaps we want a separate table for tracking versioning for the control root?

Reply:
I do like the simplicity of this table, so I'm hesitant to complicate it with the very technical implementation detail of the control root. 

On the other hand, as a user, I'd like to have a way to confirm that the control root being used corresponds to the set of `zkr` programs I trust.

Reply:
Best way for someone to check this is to look at the verified source code for the contract on Etherscan, pull the control root out of there, and verify it by recomputing it from the `risc0` repo. We should indeed include instructions on how to do this, probably in a comment above the control root field in the contract source, and potentially elsewhere.